website/content/docs/commands/curl.mdx

```diff
@@ -0,0 +1,54 @@
+---
+layout: docs
+page_title: 'Commands: curl'
+description: |
+  curl is a utility command for accessing Nomad's HTTP API similar to the
+  popular open source program of the same name.
+---
+
+# Command: curl
+
+The curl command allows easy access to Nomad's HTTP API similar to the popular
+[curl] program. Nomad's curl command reads [environment variables][envvars] to
+dramatically ease HTTP API access compared to trying to manually write the same
+command with the third party `curl` command.
+
+For example for the following environment:
+
+```
+NOMAD_TOKEN=d4434353-c797-19e4-a14d-4068241f86a4
+NOMAD_CACERT=$HOME/.nomad/ca.pem
+NOMAD_CLIENT_CERT=$HOME/.nomad/cli.pem
+NOMAD_CLIENT_KEY=$HOME/.nomad/client-key.pem
+NOMAD_TLS_SERVER_NAME=client.global.nomad
+NOMAD_ADDR=https://remote.client123.internal:4646
+```
+
+Accessing Nomad's [`/v1/metrics`][metrics] HTTP endpoint with `nomad curl`
+would require:
+
+```
+nomad curl /v1/metrics
+```
+
+Performing the same request using the external `curl` tool would require:
+
+```
+curl \
+  --cacert "$HOME/.nomad/ca.pem" \
+  --cert "$HOME/.nomad/client.pem" \
+  --key "$HOME/.nomad/client-key.pem" \
+  --connect-to "client.global.nomad:4646:remote.client123.internal:4646" \
```

Comment:
To be a fair comparison I should probably reference the environment variables when possible.

`--connect-to` is the really difficult one as it requires plucking the actual address out of `$NOMAD_ADDR`. Not impossible, but not trivial interpolation like everything else.

Still... this entire Go command could be replaced by a fairly trivial `bash` function for people to include in their profiles similar to autocompletion. Obviously that would require `bash` (not guaranteed on Windows)  and `curl` to be installed.